components/views/dashboard/components/usersOverview/index.tsx

```diff
@@ -8,7 +8,7 @@ const UsersOverview = () => {
 
     const popoverContent = (
         <ul>
-            {users.map((user: any, index: number) => (
+            {users?.users?.map((user: any, index: number) => (
```

Comment:
Dear god. why and how does this work for you? Set it back to the 1 layer of `user` and make sure this branch is up to date with `UI-1.3.0` so the redux store is accurately storing the user data